src/js/components/Box/stories/Round.js

```diff
@@ -52,6 +52,22 @@ export const RoundBox = () => (
           left rounded corner px value
         </Box>
       </Grid>
+      <Grid columns="small" gap="small">
+        <Box
+          background={{ color: 'background', dark: true }}
+          pad="small"
+          round={{
+            corners: {
+              'top-left': 'small',
+              'top-right': 'large',
+              'bottom-left': 'medium',
+              'bottom-right': 'xsmall',
+            },
+          }}
+        >
+          <Text color="brand">Individually rounded corners</Text>
```

Comment:
```suggestion
          <Text>Individually rounded corners</Text>
```
Currently there isn't enough contrast between the box background (black) and the text color (purple) in the storybook example. Can we remove the color assignment on the text and then it will default to white text on a dark background

Reply:
Sure.